py_hamt/hamt_to_sharded_converter.py

```
@@ -0,0 +1,130 @@
+    args = parser.parse_args()
+    # Initialize the KuboCAS client with the provided RPC and Gateway URLs
+    async with KuboCAS(
+        rpc_base_url=args.rpc_url, gateway_base_url=args.gateway_url
+    ) as cas_client:
+        try:
+            await convert_hamt_to_sharded(
+                cas=cas_client,
+                hamt_root_cid=args.hamt_cid,
+                chunks_per_shard=args.chunks_per_shard,
+            )
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)